decred/decred/dcr/dcrdata.py

```diff
@@ -318,13 +318,13 @@ def makeOutputs(pairs, chain):
     """
     makeOutputs creates a slice of transaction outputs from a pair of address
     strings to amounts.  This is used to create the outputs to include in newly
-    created transactions from a JSON object describing the output destinations
-    and amounts.
+    created transactions from a JSON-derived dict describing the output
```

Comment:
Everytime I typed "JSON object", I cringed a little inside, but I was too lazy to think of a better description. This works.